### DISCUSSION TITLE
Feature Request:  utils command or flag for checking if generated mardown docs have changed.

### DISCUSSION THREAD

**Discussion by sullivancolin** [tiangolo/typer-cli#139] [Questions] [answered]
I'm not sure it this functionality belongs in typer-cli or should be implemented outside of it.

Basically, I'm interested in similar behavior to `black --check` or `isort --check` which can be run in CI or pre-commit to warn the developer that they need to run those tools.   I have auto generated docs and I want them to be kept in sync with code changes to the CLI and trigger a CI failure if the previous version of the docs no longer matches what would be generated now.  It wouldn't be too hard to implement this outside of typer-cli with a bash script, but curious if others would find it useful.

**Comment by tiangolo** [comment, marked as answer]
Hello! Now Typer-CLI is part of Typer, when you install `typer` you get the `typer` command too.

About your question, it would be a boolean option, and then the internal code you run would read it and alter its behavior: https://typer.tiangolo.com/tutorial/parameter-types/bool/
